mito-ai/mito_ai/utils/telemetry_utils.py

```diff
@@ -278,3 +278,22 @@ def log_ai_completion_success(
 
         log(f"mito_ai_{message_type.value}_success", params=final_params, key_type=key_type, thread_id=thread_id)
         
+def log_db_connection_attempt(connection_type: str) -> None:
+    log("mito_ai_db_connection_attempt", params={"connection_type": connection_type})
+
+def log_db_connection_success(connection_type: str, schema: dict[str, Any]) -> None:
+    log(
+        "mito_ai_db_connection_success", 
+        params={
+            "connection_type": connection_type, 
+            }
+        )
+
+def log_db_connection_error(connection_type: str, error_message: str) -> None:
```

Comment:
It would be nice for these to still log when the user is on the pro + mito_server because for initial rollout, I think that most users will be pro people we do testing with. However, right now because we are not passing the key_type to the log function, we don't turn on telemetry for pro users. 

I don't know the best way to fix this though because of how we've set up our backend. The providers.py file is where we know about the key_type and the db connection doesn't go through that at all. 

Take a look at how `log_ai_completion_success` for more context on how this works. 

Any quick solutions? Lmk if you need more context.